src/components/UploadCSV/UploadCSVUtils.jsx

```diff
@@ -24,7 +24,7 @@ export default yup
   .object({
     boxNumber: yup.number().boxNotExists().required().typeError('Missing or invalid box number'),
     date: yup.date().required().typeError('Missing or invalid date'),
-    zipCode: yup.string().isZip().required('Missing or invalid zip code'),
+    zipCode: yup.string().isZip().required('Missing zip code'),
```

Comment:
If I recall correctly you can also do this: 
`yup.string().isZip('Invalid Zip Code').required('Missing zip code')`
Just suggesting that since in the last commit the error comment was 'Missing or invalid zip code'